Add new function

Defman21 opened the pull request:
Ask for user input (command for run).
If custom command is empty - run default (ls -la) command.
Additional info:
typeof(command) = string.

Defman21 committed:
Add new function

Ask for user input (command for run).
If custom command is empty - run default (ls -la) command.

th3coop commented:
Hey Sergey, can you make two changes. 
1.  Don't remove the space from the thrown error at line 26.
2.  change the ternary operation to an "if" one-liner so you don't have to re-assign command to command
eg ```if (command.trim() == "") command = "ls -la";`

Defman21 committed:
Change what cgchoffman ask

Return space on line 26.
Remade "command" condition (remove "else", not needed).

Defman21 committed:
Oneline condition style

Oops, forgot that I must create a one-line condition :)

Defman21 committed:
remove some brackets

I'm stupid :D
Brackets on line 34 not need :)

Defman21 commented:
Well, I do what you want.
